src/index.ts

```diff
@@ -24,20 +24,29 @@ export const createApi = (gitlabToken?: string) => {
   }
 
   const token = gitlabToken || process.env.GITLAB_TOKEN
+  if (!token) {
+    throw new Error('GitLab token is not set')
+  }
```

Comment:
I'm not sure if I should throw the error here. It seems like `@gitbeaker/node` accepts `undefined` as the token but it's not the case with `@gitbeaker/rest`.

Reply:
`GITLAB_TOKEN` env is marked as required in README, so we can just assert `process.env.GITLAB_TOKEN` can never be nil here.

Reply:
Would it be better if we export an `env` object from a module that has defined the "actual" type of the environment variables? I could open another PR to make this change afterward. 😄

Reply:
> Would it be better if we export an `env` object from a module that has defined the "actual" type of the environment variables? I could open another PR to make this change afterward. 😄

Sure!